spec/namespaces/insights.yaml

```diff
@@ -0,0 +1,34 @@
+openapi: 3.1.0
+info:
+  title: Query Insights API
+  description: API to retrieve top queries based on latency, CPU, or memory usage.
+  version: 1.0.0
+paths:
+  /_insights/top_queries:
+    get:
+      operationId: insights.top_queries.0
+      x-operation-group: insights.top_queries
+      x-version-added: '1.0'
```

Comment:
I think this should be 2.15?

Reply:
yes. Let me send out a follow up on this.